src/status_im/subs/root.cljs

```diff
@@ -169,7 +169,7 @@
 (reg-root-key-sub :wallet-connect/web3-wallet :wallet-connect/web3-wallet)
 (reg-root-key-sub :wallet-connect/current-proposal :wallet-connect/current-proposal)
 (reg-root-key-sub :wallet-connect/current-request :wallet-connect/current-request)
-(reg-root-key-sub :wallet-connect/pairings :wallet-connect/pairings)
+(reg-root-key-sub :wallet-connect/persisted-sessions :wallet-connect/persisted-sessions)
```

Comment:
should be called "active-sessions" IMO. Don't think we should differentiate between the persisted ones and the ones we get from web3-wallet cause the persisted ones are just a fallback

Reply:
`persisted-sessions` might be different from active sessions as the user can disconnect from the dapp when status-app is closed. 

I introduced this term `persisted-sessions` in the UI when I wrote the handlers for saving and retrieving sessions.

Reply:
I've replaced it with just `sessions`, because the list consists of persisted sessions first and then get replaced with active sessions once they are loaded from the SDK (which means it would be inaccurate to always call it `active sessions`)

Reply:
@shivekkhurana indeed they're different, but what we're storing in the db is not exactly the persisted sessions, but rather the sessions that are active. It's more of an implementation detail that we're falling back on the persisted-sessions when there's no internet connection.